.github/workflows/argilla.docs.yml

```diff
@@ -55,16 +57,17 @@ jobs:
         run: echo "${{ github.ref }}"
           echo "${{ github.head_ref }}"
 
+      - run: |
+          pdm run mike deploy latest --push
+          pdm run mike set-default latest
```

Comment:
```suggestion
          pdm run mike deploy develop --push
```
I would maybe just call the latest version of the docs/develop. I think we should try avoid the flow you had because merging new feature into main might then cause in doc updates without the supported feature.